topcoffea/modules/corrections.py

```diff
@@ -498,6 +498,17 @@ def ApplyJetSystematics(year,cleanedJets,syst_var):
   elif(syst_var == 'nominal'): return cleanedJets
   elif(syst_var in ['nominal','MuonESUp','MuonESDown']): return cleanedJets
   # Save `2016APV` as `2016APV` but look up `2016` corrections (no separate APV corrections available)
+  elif('Flavor' in syst_var and 'Up' in syst_var and syst_var.replace('Up', '') in cleanedJets.fields or 'Down' in syst_var and syst_var.replace('Down', '') in cleanedJets.fields):
+      if 'Gluon' in syst_var: mask = abs(cleanedJets.partonFlavour)==21
+      elif 'Bottom' in syst_var: mask = abs(cleanedJets.partonFlavour)==5
+      elif 'Charm' in syst_var: mask = abs(cleanedJets.partonFlavour)==4
+      elif 'Quark' in syst_var: mask = abs(cleanedJets.partonFlavour)<4
```

Comment:
Should we have an `else` here? I'm wondering if there is a case where you expect something besides `Gluon`, `Bottom`, `Charm`, or `Quark` in the name? If yes, what would we want to set the mask equal to in that case? If no, then can we raise an exception here (just to make sure nothing unexpected is going on)?

Reply:
No, there's nothing else in the JES files. I can add an exception for bug checking.